frameworks/person-escort-record/README.md

```diff
@@ -16,6 +16,7 @@ Manifest files should live in [`manifests/`](./manifests) and be stored in a fla
 
 - `name` **(required)** - name of the section. Used for the step heading caption.
 - `steps` - a list of steps for this section. See [steps documentation](#steps) for options.
+- `order` - order in which to display the sections in summaries or a task list. Defaults to alphabetically when no order is provided.
```

Comment:
I would avoid adding a default, as it depends really on how this framework is consumed, so maybe make this required

Reply:
A tricky one because this is really only for our frontend I guess. And no different to how we've defined other things like `hint` being the text shown under a question - if it's consumed by something else that could also not be strictly true like in this sense...

Reply:
Removed the default 👍